doc/Development_Documentation/23_Installation_and_Upgrade/07_Updating_Pimcore/10_V6_to_V10.md

```diff
@@ -47,6 +47,19 @@ Important changes:
 - Move templates from `app/Resources/views` to `templates` and bundles templates from `app/Resources/XYZBundle/views` to `templates/bundles/XYZBundle`
 - Move `app/AppKernel.php` to `src/Kernel.php` and edit it to have `namespace App;` and class name `Kernel`
 - Rename `web/` to `public/` & `web/app.php` to `public/index.php`
+- Rename `config/routing.yml` to `config/routes.yaml`
+- Move your Environment specific configs to `config/packages/$ENV/'.yaml`
```

Comment:
Something is not right here... (`'` as filename?)

Reply:
sorry there was a typo issue.
thx for the info 👍